ibtk/src/math/fortran/div2d.f.m4

```diff
@@ -312,6 +312,65 @@ c
 c
       return
       end
+ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+c
+c     Computes D = alpha div (c u).
```

Comment:
Suggest to make the comments match the interface --- if it is `div (c u)` I think that the arguments in the function should be in that order also (or vice versa).

Of course, in the implementation, the order does not matter, and so this also would be fine to describe as something like `div u v`.